spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB Table' do
```

Comment:
Please write a spec for a working factory

Reply:
There's now a spec for a valid factory